framework/auth/views.py

```diff
@@ -49,6 +50,10 @@ def reset_password_get(auth, uid=None, token=None):
     :raises: HTTPError(http.BAD_REQUEST) if verification key for the user is invalid, has expired or was used
     """
 
+    #override routes.py login_url to redirect to dashboard
+    next_url = re.sub(r'resetpassword/[A-z,0-9,/]+', '', request.url)
+    next_url = next_url + 'dashboard/'
+    service_url = cas.get_login_url(next_url)
```

Comment:
Similarly, no need to replace substrings here, you can just use `web_url_for` - with `_absolute`, the domain will be correct.  Also, I'd locate this lower, as it's not needed until you're returning a response.